spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -192,8 +195,6 @@ export function connectToSpotTV(joinCode, shareMode) {
 
             _clearSubscriptions();
 
-            dispatch(clearSpotTVState());
```

Comment:
why does it go away exactly? is it to not drop out of the remote control view? is there another place where this cleanup will happen?

Reply:
This is already called on line 171. I think this is a duplicate call to clearSpotTVState.

Reply:
Gonna add it back. If we want to remove it then we should do it at some other time. Keep this PR focused on the reconnect logic only.